models/MITgcm_ocean/model_mod.f90

```diff
@@ -533,11 +522,7 @@ subroutine static_init_model()
 if (do_output()) write(     *     , *) 'Using grid size : '
 if (do_output()) write(     *     , *) '  Nx, Ny, Nz = ', Nx, Ny, Nz
 
-call parse_variable_input(mitgcm_variables, model_shape_file, nvars, &
-                      var_names, quantity_list, clamp_vals, update_list)
-
-domain_id = add_domain(model_shape_file, nvars, &
-                    var_names, quantity_list, clamp_vals, update_list )
+domain_id = add_domain(model_shape_file, parse_variables_clamp(mitgcm_variables))
```

Comment:
I don't think you can replace the routine for MITgcm_ocean because the original code adjusts the clamping values if you are doing a log transform for BGC variables:

https://github.com/NCAR/DART/blob/f2deaa27499fea386d4b72d44a4134f4adf2ab62/models/MITgcm_ocean/model_mod.f90#L2113-L2123